git-publish

```diff
@@ -449,14 +449,25 @@ def menu_select(menu):
             continue
         return a
 
+def edit_content(content, suffix):
+    fd, tmpfile = tempfile.mkstemp(text=True, suffix=suffix)
+    fd_opened = os.fdopen(fd, 'wb')
```

Comment:
Why is binary I/O with manual encoding/decoding necessary? Python's text I/O mode should work, so maybe the issue was that ENCODING has the wrong value on your system?